src/components/data-display/tables/DomainsTable.tsx

```diff
@@ -346,14 +315,9 @@ const DomainsTable = ({
                 <button
                   className="flex whitespace-nowrap justify-center align-center gap-2 text-center"
                   onClick={async () => {
-                    dispatchANTUpdate({
-                      processId,
-                      domain: lowerCaseDomain(row.original.name),
-                      aoNetwork,
-                      queryClient,
-                      walletAddress,
-                      hyperbeamUrl,
-                      dispatch: dispatchArNSState,
+                    await queryClient.invalidateQueries({
+                      queryKey: ['ant', processId],
```

Comment:
This is a less dependant way to indicate the query key - omitting 'ant' will also refresh _any_ apis using the process id.

Query key is position sensitive, so if processId is changed in the future to another index this breaks. Using the predicate helps future proof a bit there.

```suggestion
                    await queryClient.invalidateQueries({
                      predicate: ({ queryKey }) => queryKey.includes("ant") && queryKey.includes(processId),
```

Reply:
Another note there, check in with the new hooks on things relying on state... I believe the useAntsForWallet hook calls this in its query, but its state may not be updated in this refresh.

Thats a case where including the process id in the query key and then predicating on the process ID can help in cache control.